CONTRIBUTING.md

```diff
@@ -113,18 +87,7 @@ To publish a new version of updated packages, please make sure you:
 - have push rights to this repository's `main` branch
 - are on a clean `main` branch
 
-Follow these steps to publish the updated packages:
-
-- Build and test from root
-- Bump the version
-- Publish updated packages
-
-```bash
-rush build
-rush test
-rush version --bump -b main
-rush publish --apply --publish --include-all --add-commit-details --set-access-level public --target-branch main
-```
+TODO
```

Comment:
Let's pick this up later on (w/ changesets). Noticed much of publishing we do is pretty manual so not much harm done by removing this process atm I guess.